### PR TITLE
[CS2103-T14-2] T.A.rence

### PR DESCRIPTION
**Pull request by Ellieyee**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -295,72 +295,352 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
 
 [appendix]
 == Non Functional Requirements
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+=== Business/Domain Rules:
+. System should be able to handle multiple modules and tutorials
+. Number of module and tutorial slots created should be of reasonable amount (< 10)
```
Comment (joannasara) [inline docs/DeveloperGuide.adoc:595]: I think this is a limitation instead.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -295,72 +295,352 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+=== Performance Requirements:
+. System should be able to hold up to 100 students without a noticeable sluggishness in performance for typical usage.
+
+=== Quality Requirements:
+. A user with above average typing speed for regular English text (i.e. not code, not system admin commands)
+should be able to accomplish most of the tasks faster using commands than using the mouse.
+. The application should work even if the user enters partial commands/data and data in
+different formats, or omits certain command prefixes.
+. Product should be usable by someone who just became a Teaching Assistant.
```
Comment (joannasara) [inline docs/DeveloperGuide.adoc:613]: You might want to clarify why being a new Teaching Assistant might be a problem.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -295,72 +295,352 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
```
Comment (victorvic54) [inline docs/DeveloperGuide.adoc:296]: Probably can specify the adding of module here

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
...
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: manage TA-related tasks faster than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
```
Comment (victorvic54) [inline docs/DeveloperGuide.adoc:291]: Can add the adding tutorial in user stories

**Review by joannasara — COMMENTED** [review]
Such a comprehensive Developer Guide!

**Review by victorvic54 — COMMENTED** [review]
Overall clear user stories. Good job!

**Comment by damithc** [comment]
Corrected wrong PR name. You are in CS2103, not CS2103T

**Pull request closed without merging**